### PR TITLE
Initial Refactoring

### PR DESCRIPTION
**Pull request by pankiljoshi**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,8 @@
         }
     },
     "scripts": {
-        "tests": "vendor/bin/phpunit"
+        "tests": "vendor/bin/phpunit",
+        "dynamodblocalstart": "sudo docker run -p 8000:8000 amazon/dynamodb-local"
```
Comment (jdecode) [inline composer.json]: @pankiljoshi - Don't add more responsibility than required.
DynamoDB (local or online) should be considered as working (failed test cases would let us know that the DynamoDB isn't running).
Also, never use "sudo"!

**Comment by jdecode** [comment]
@pankiljoshi  - Is there any new functionality added or this is refactoring only?

File: composer.json (hunk 1 above)
Comment (pankiljoshi) [reply composer.json]: Noted!!

**Comment by pankiljoshi** [comment]
@jsucreate earlier commits were refactoring only, new commit has "update" method.

**Comment by jdecode** [comment]
My GitHub id is "jdecode" not "jsucreate" - that one is of Jatinder Singh.

@pankiljoshi - If it is only refactoring, then never change test cases.
Changing test case means change in functionality.
Unfortunately, I won't be able to accept this PR.
Please cancel it and create a new one - focus on the refactoring without changing test cases.

Then create a separate commit with update method's test cases and it's respective code.

**Comment by pankiljoshi** [comment]
@jdecode - Closed it, so what should be the approach if in my opinion those changes can be beneficial for the package?

**Comment by jdecode** [comment]
@pankiljoshi -:
Refactor without changing test cases : This will confirm that the refactor doesn't breaks something.

For new functionality - write test cases first and then write code to "just" to pass the test cases.

Now you would have :
1. Refactored existing code (without changing functionality, by not changing test cases)
2. Write new test cases
3. Write code to "just" pass the test cases (code quality doesn't matter yet)
4. Now you refactor (again, without changing test cases) - and keep improving the code quality.

**Pull request closed without merging**